packages/react-jss/src/JssContext.js

```diff
@@ -2,9 +2,7 @@
 import React, {type Context} from 'react'
 import type {Context as JssContextValue} from './types'
 
-const JssContext: Context<JssContextValue> = React.createContext({
-  sheetOptions: {},
+export const JssContext: Context<JssContextValue> = React.createContext({
```

Comment:
Why do we change the default export to a named one?

Reply:
Fixed